### ISSUE TITLE
GPX export from OsmAnd: Times are without TZ indicator

### ISSUE THREAD

**Issue by otdftr** [osmandapp/OsmAnd-iOS#1809]
GPX-export from 4.1.1 on iOS 15.3.3: <time>..</time> values are without timezone - time is local time (in this case CET):
````
    <metadata>
        <name>2022-03-11_14-14_Fr.</name>
        <time>2106-02-07T07:28:15</time>
    </metadata>
    <trk>
        <trkseg>
            <trkpt lat="51.0897720" lon="6.8997210">
                <ele>42.8</ele>
                <time>2022-03-11T14:14:17</time>
                <hdop>0.000000000000</hdop>
                <extensions>
                    <speed>0.000</speed>
                </extensions>
            </trkpt>
            <trkpt ...
````

This puzzles other apps, in my case
- MyTourbook will not import the file due to "invalid / unparseable timestamps"
- GPS RouteConverter will import the file but will assume UTC / Zulu TZ, effectively moving the time 1 hour forward

Yes, TZ indicator is optional in ISO-8601, and one could blame the importing apps for not adhering to the standard.
Nevertheless, timestamps in the OsmAnd's iOS export currently are ambiguous, as neither file nor header carry TZ information.

I would like to suggest this for improvement / mitigation:

IMHO the GPX file should contain unambiguos timestamps, preferably UTC timestams w/ "Z" indicator: `2022-03-11T13:14:17Z` (as, BTW, the Android app does). To me, this seems to be 'more' portable than timestamps w/ local time + proper TZ offset `2022-03-11T14:14:17+0100`.

**Comment by vshcherb** [comment]
Already fixed in 4.2, available in Test Flight

**Comment by dmpr0** [comment]
You can get TestFlight by following the link https://testflight.apple.com/join/7poGNCKy
